CMakeLists.txt

```
@@ -584,7 +584,17 @@ if (YUZU_USE_BUNDLED_FFMPEG)
             CACHE PATH "Path to FFmpeg headers" FORCE)
 
         # `configure` parameters builds only exactly what yuzu needs from FFmpeg
-        # `--disable-{vaapi,vdpau}` is needed to avoid linking issues
+        # `--disable-vdpau` is needed to avoid linking issues
+
+        if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+            pkg_check_modules(LIBVA REQUIRED)
+            pkg_check_modules(LIBVA-DRM REQUIRED)
+            pkg_check_modules(LIBVA-X11 REQUIRED)
```

libva-x11 requires x11 but it is not linked.

```suggestion
            pkg_check_modules(LIBVA libva)
            pkg_check_modules(LIBVA_DRM libva-drm)
            pkg_check_modules(LIBVA_X11 libva-x11)
            pkg_check_modules(X11 x11)
```

Then use `<prefix>-FOUND` and `<prefix>-LIBRARIES` as appropriate.

---

So that's why LIBVA_LIBRARIES wasn't working for me before.

Fixed.